File: tecnonet-angular/src/app/models/admin-dashboard.dto.ts
import { Contrato } from "./contrato.model";
import { Usuario } from "./usuario.model";

export interface AdminDashboardDTO {

    ingresosDelMes: number;
    totalClientesActivos: number;
    totalContratosActivos: number;
    solicitudesPendientes: number;

    nuevosClientes: Usuario[];
    contratosPendientes: Contrato[];
}

File: tecnonet/src/main/java/com/ProyectoTecnonet/tecnonet/dto/AdminDashboardDTO.java
package com.ProyectoTecnonet.tecnonet.dto;

import java.math.BigDecimal;
import java.util.List;

import com.ProyectoTecnonet.tecnonet.model.Contrato;
import com.ProyectoTecnonet.tecnonet.model.Usuario;

import lombok.Data;

@Data
public class AdminDashboardDTO {

    private BigDecimal ingresosDelMes;
    private long totalClientesActivos;
    private long totalContratosActivos;
    private long solicitudesPendientes;

    private List<Usuario> nuevosClientes;
    private List<Contrato> contratosPendientes;
}